File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/membrii-afiliati-report-row.model.ts

import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";

@JsonObject
export class MembriiAfiliatiReportRowModel {
    
	@JsonProperty("institutie", String)
    public institutie: string = null;
    
	@JsonProperty("reprezentant", String)
    public reprezentant: string = null;
    
	@JsonProperty("comisie", String)
    public comisie: string = null;
     
    @JsonProperty("dataSedinta", JsonDateConverter)
    public dataSedinta: Date = null;
    
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/MembriiAfiliatiReportRowModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.Date;

public class MembriiAfiliatiReportRowModel {

	private String institutie;
	private String reprezentant;
	private String comisie;
	private Date dataSedinta;

	public String getInstitutie() {
		return institutie;
	}

	public void setInstitutie(String institutie) {
		this.institutie = institutie;
	}

	public String getReprezentant() {
		return reprezentant;
	}

	public void setReprezentant(String reprezentant) {
		this.reprezentant = reprezentant;
	}

	public String getComisie() {
		return comisie;
	}

	public void setComisie(String comisie) {
		this.comisie = comisie;
	}

	public Date getDataSedinta() {
		return dataSedinta;
	}

	public void setDataSedinta(Date dataSedinta) {
		this.dataSedinta = dataSedinta;
	}

}
